stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.scss

```diff
@@ -111,17 +111,7 @@
       }
     }
 
-    .icon-bar {
-      display: none;
-
-      @include devicebreak(mediumonlytab) {
-        display: block;
-      }
-    }
-
-    .icon-close {
-      display: none;
-    }
+    @extend %header-close;
```

Comment:
@Ashish-srij Change the placeholder name to just `menu-icon`.

Reply:
@shishirphadke Done.